DEVELOPMENT.md

```diff
@@ -1,134 +1,87 @@
+    $ poerty lock
+    $ poetry install
+    
+first bofore running
+    
     $ nix develop
 
-to build the environment more rapidly. Note that this only helps on
-the first occasion that `nix develop` is executed. `nix develop`
-should be near instantaneous on subsequent runs regardless of whether
-Cachix is used.
-
-To upload a new PFHub environment to Cachix use
+and then run the CLI with
 
-    $ cachix authtoken <TOKEN>  # get from cachix.org
-    $ nix develop --profile pfhub-profile -c true
-    $ cachix push pfhub pfhub-profile
+    $ pfhub --help
+    
+To run the CLI directly use
 
-## Update Flakes
+    $ nix run .#pfhub -- --help
 
-The flake can be updated to have new package versions using first
+## Additional odds and ends
 
-    $ nix flake lock --update-input pfhub
-
-and then
-
-    $ nix flake update
-    $ nix develop
+### Nix Shell Prompt
 
-The initial command is required since the base `flake.nix` depends on
-`_data/python-pfhub/flake.nix` and issues can occur if the dependency
-is not also updated. See [this issue on
-GitHub](https://github.com/NixOS/nix/issues/3978#issuecomment-1585001299).
-
-When the flake is updated, a new `flake.lock` file is generated which
-must be committed to the repository alongside the `flake.nix`. Note
-that the flake files depend on the `nixos-23.05` branch which is only
-updated with backports for bug fixes. To update to a newer version of
-Nixpkgs then the `flake.nix` file requires editing to point at a later
-Nixpkgs version.
-
-> **NOTE**
-> Jekyll seems to be broken in Nixpkgs 23.05 so 22.11 is used for only
-> thi package. Hence why the flake depends on two versions of Nixpkgs.
-
-
-# Install PFHub using Nix
-
-## Installation
-
-PFHub is developed using [Nix](https://nixos.org/) and Nix is
-recommended for running and using the functionality that PFHub
-provides. Nix should be easy to install, but some problems can
-occur. Please raise an
-[issue](https://github.com/wd15/pfhub/issues/new?assignees=&labels=&projects=&template=blank.md)
-if the following isn't clear or you get stuck.
-
-Here are some sites to help with installation.
-
- - [Official download](https://nixos.org/download.html)
- - [Official documentation](https://nix.dev/tutorials/install-nix)
- - [Tutorial](https://nix-tutorial.gitlabpages.inria.fr/nix-tutorial/installation.html)
- - [My own notes if paths are
-   missing](https://github.com/wd15/nixes/blob/master/NIX-NOTES.md)
- - [Another tutorial](https://gricad.github.io/calcul/nix/tuto/2017/07/04/nix-tutorial.html#install-nix-single-user-mode)
-
-Once Nix is installed try it out with:
-
-    $ nix-env -i hello
-    installing 'hello-2.12.1'
-    $ ./hello
-    Hello, world!
-
-
-> **NOTE**
-> The [PFHub Dockerfile](./Dockerfile) installs Nix on Ubuntu
-> 22.04 so might be a place to consult to get paths and the like setup correctly.
-> See the [PFHub Docker usage instructions](./DOCKER.md).
+**NOTE**: the `nix develop` command fails to change the shell prompt
```

Comment:
@guyer is there a macos/`zsh` version of this?

Reply:
How the heck would I know? I don't zsh.

Reply:
Ah. My understanding is that `zsh` is the default for macos, hence the question.

Reply:
It is, but if you've been using a Mac since 1984, they don't force you to switch

Reply:
Forcing decisions on you is what makes a mac a Mac! Steve Jobs is rolling in his grave.